packages/next/server/api-utils.ts

```diff
@@ -63,6 +63,26 @@ export async function apiResolver(
       )
     }
 
+    let contentLength = 0
+    const writeData = apiRes.write
+    const endResponse = apiRes.end
+    apiRes.write = (...args) => {
+      contentLength += Buffer.byteLength(args[0])
+      writeData.apply(apiRes, args)
+    }
+    apiRes.end = (...args) => {
```

Comment:
```suggestion
    apiRes.end = (...args: any[]) => {
```
I think it's alright to use any here since it can have various types and we're just wrapping here